NSS-Spontaneous-Spoonbills/NSS-Spontaneous-Spoonbills
# Patrick - added employee/department/training_program modules

Added Serializers and views for three tables:
Employee
Department
Training_Program

Updated the README
Steps to Test
Run virtual environment
Start django app
Navigate to http://127.0.0.1:8000/bangazonapi/[tablename]
There are three tables to test
Verify that you can see the data from the database

```
git fetch --all
git checkout patrick
```

Impacted Areas in Application
##### Modified
	modified:   bangazonapi/serializers/Department_Serializer.py
	modified:   bangazonapi/serializers/Employee_Serializer.py
	modified:   bangazonapi/serializers/Employee_Training_Serializer.py
	modified:   bangazonapi/serializers/Product_Serializer.py
	modified:   bangazonapi/serializers/Training_Program_Serializer.py
	modified:   bangazonapi/serializers/User_Serializer.py
	modified:   bangazonapi/serializers/__init__.py
	modified:   bangazonapi/urls.py
	modified:   bangazonapi/views/Department_View.py
	modified:   bangazonapi/views/Employee_Training_View.py
	modified:   bangazonapi/views/Employee_View.py
	modified:   bangazonapi/views/Product_View.py
	modified:   bangazonapi/views/Training_Program_View.py
	modified:   bangazonapi/views/User_View.py
	modified:   bangazonapi/views/__init__.py

##### Deleted
	deleted:    bangazonapi/views/User_View.py~HEAD

##### to be Tracked

Mentions @username
@MrErin @obscure-shadow @JacobAndrewSmith92 



Definition of Done
The project must be fully documented. This includes the following:
Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
Every class must be documented with purpose, author, and methods.
Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
The project must be able to run fully, and without errors, on each teammate's system.
Fulfills every requirement.
Every line of code has been peer reviewed.
For projects that require unit testing, core functionality must be identified and have at least one test for each
